docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Commencing
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y vim=2:8.0.1453-1ubuntu1
+RUN apt-get install -y openssl
+RUN apt-get install -y libssl-dev
+RUN apt-get install -y libcurl4-openssl-dev
+RUN apt-get install -y git gpg
+RUN apt-get install -y libomp-dev
+RUN apt-get install -y gsl-bin libgslcblas0 libgsl23 libgsl-dev libxml2-dev
+RUN apt-get install -y python3-venv python3-pip python3-dev build-essential cmake
+RUN pip3 install --upgrade pip
+
+
+### Install R
+
+#httpv:/ cran.rs.udio.com/bin/linux/ubuntu/
+#RUN sh -c 'echo "deb http://cran.rstudio.com/bin/linux/ubuntu/bionic-cran35/" >> /etc/apt/sources.list'
+RUN sh -c "echo \"deb http://cran.rstudio.com/bin/linux/ubuntu bionic-cran35/\" >> /etc/apt/sources.list"
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
+RUN apt-get -y update
+RUN apt-get -y install r-base
+RUN apt-get -y install r-base-dev
+RUN apt-get -y install r-recommended
+
+
+
+
+
+
+
+#### Py-RerF
+RUN pip3 install -e "git+https://github.com/neurodata/RerF.git@staging#egg=rerf&subdirectory=Python"
+
```

Comment:
to run the sklearn classifier example we need pandas

`pip3 install pandas`

it's not in the requirements.txt file since it's not strictly a dependency, but we should probably add it here in the dockerfile for convenience